spec/workers/pubsubhubbub/delivery_worker_spec.rb

```diff
@@ -10,7 +10,13 @@
 
   describe 'perform' do
     it 'raises when subscription does not exist' do
-      expect { subject.perform 123, payload }.to raise_error(ActiveRecord::RecordNotFound)
+      expect(
+        begin
+          subject.perform 123, payload
+        rescue Mastodon::WorkerError => e
+          e.cause
+        end
+      ).to be_an(ActiveRecord::RecordNotFound)
```

Comment:
I think there is better way to write test to check e.cause... but I didn't found it..